framework/test/grt-eval-tab4-parallel.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+################################################################################
+#
+# This script runs grt-eval-tab4.sh in parallel on the Cartesian product of
+# CLASSES, GENERATORS, and TIMES defined below.
+#
+# The degree of parallelism is hard-coded as NUM_CORES. Modify this as needed.
+#
+################################################################################
+
+
+# Define parameters
+CLASSES=("Math" "Lang" "Time" "Chart")
+GENERATORS=("evosuite" "randoop" "randoopGRTMinusDynamicTyping" "randoopGRTMinusMinCostFirst" "randoopGRTMinusMinCoverageFirst" "randoopGRT")
+TIMES=(120 300 600)
```

Comment:
These are defined in multiple files.  I suggest abstracting them out into a file that can be sourced by multiple scripts.